tests/add_fee_amount.go

```diff
@@ -112,4 +112,27 @@ func AddFeeAmount(network network.Network) {
 	amount, err := subnetATeleporterMessenger.CheckRelayerRewardAmount(&bind.CallOpts{}, fundedAddress, mockTokenAddress)
 	Expect(err).Should(BeNil())
 	Expect(amount).Should(Equal(additionalFeeAmount.Add(additionalFeeAmount, initFeeAmount)))
+
+	// Redeem the relayer reward amount
+	balanceBeforeRedemption, err := mockToken.BalanceOf(
+		&bind.CallOpts{}, fundedAddress,
+	)
+	Expect(err).Should(BeNil())
+	// TODO: figure out why the following is reverting with message
+	// "execution reverted: TeleporterMessenger: no reward to redeem"
+	transaction, err := subnetATeleporterMessenger.RedeemRelayerRewards(
+		&bind.TransactOpts{}, fundedAddress,
```

Comment:
I think this should use `mockTokenAddress` instead of `fundedAddress`. The parameter of `redeemRelayerRewards` is the fee asset contract to redeem the rewards for, with the "redeemer" address being the caller of the method. So we should be sure this transaction is being sent from the account specified by the relayer as the reward redeemer, and specify the mock token as the asset being redeemed.